committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,76 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
```

Comment:
I am fine with following the bootstrapping guide, where my seat would be 2 years instead of 1.

Reply:
It is the rules as written, even if the announcement of terms was mistaken.